6.recursiveSum.js

```diff
@@ -9,7 +9,12 @@
 
 
 const recursiveSum = (arr) => {
-
+    //implement i and update it recursively
+    let array = arr
+    if(array.length <= 1) return array[0]
+    let i = array.length - 1
+    array.pop()
+    return array[i] + recursiveSum(array)
 }
```

Comment:
This misses a spec because the directions state it needs to be pure. Therefore you can't use pop as it mutates the array.